pkg/knuu/knuu.go

```diff
@@ -99,15 +101,28 @@ func InitializeWithScope(scope string) error {
 		return ErrCannotInitializeK8s.Wrap(err)
 	}
 
-	if err := handleTimeout(); err != nil {
-		return ErrCannotHandleTimeout.Wrap(err)
+	traefikClient = &traefik.Traefik{
+		K8s: k8sClient,
+	}
+	if err := traefikClient.Deploy(ctx); err != nil {
+		return ErrCannotDeployTraefik.Wrap(err)
+	}
+
+	publicIP, err := traefikClient.Endpoint(ctx)
+	if err != nil {
+		fmt.Printf("Error getting traefik Endpoint: %v", err)
```

Comment:
This should return the error instead of just printing it.

Reply:
nice catch